x-pack/plugins/observability_solution/slo/docs/openapi/slo/components/schemas/kql_with_filters_good.yaml

```diff
@@ -0,0 +1,14 @@
+title: KQL with filters
```

Comment:
Both `kql_with_filters_good.yaml` and `kql_with_filters_total.yaml` have the same title `KQL with filters`. Is it fine, or shall we maybe differentiate it a bit?

Reply:
Good point, I've updated the text in https://github.com/elastic/kibana/pull/186575